src/components/CandidateBox.js

```diff
@@ -0,0 +1,49 @@
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import Candidate from './Candidate';
+import axios from 'axios';
+import { url } from './url';
+
+const CandidateBox = () => {
+  const [voteCandidate, setVoteCandidate] = useState('');
+  const getCandidates = async () => {
+    try {
+      const Candidates = await axios.get(`${url}/candidates`);
+      setVoteCandidate(Candidates.data);
+    } catch (e) {
+      console.error(e);
+    }
+  };
+  useEffect(() => {
+    getCandidates();
+  }, []);
+
+  return (
+    <Wrapper>
+      {voteCandidate &&
+        voteCandidate
+          .sort((a, b) => {
+            return b.voteCount - a.voteCount;
+          })
```

Comment:
화살표 함수에서는 이렇게 중괄호와 return문을 생략할 수도 있어요~

```suggestion
          .sort((a, b) =>b.voteCount - a.voteCount;)
```

Reply:
앗 그런 방법이!!